build/common.props

```diff
@@ -3,7 +3,7 @@
     <!-- Packages can have independent versions and only increment when released -->
     <Version>3.0.29$(VersionSuffix)</Version>
     <ExtensionsStorageVersion>4.0.5$(VersionSuffix)</ExtensionsStorageVersion>
-    <HostStorageVersion>4.0.2$(VersionSuffix)</HostStorageVersion>
+    <HostStorageVersion>4.1.0$(VersionSuffix)</HostStorageVersion>
```

Comment:
@fabiocav @mathewc given the reduced changes, should this still be 4.1.0?

Reply:
Yes, 4.0.3 is probably better - following our regular patch versioning.